m3u8/model.py

```diff
@@ -959,7 +959,10 @@ def __init__(self, uri, stream_info, media, base_uri):
         if resolution is not None:
             resolution = resolution.strip('"')
             values = resolution.split("x")
-            resolution_pair = (int(values[0]), int(values[1]))
+            resolution_pair = (
+                int(round(float(values[0]))),
```

Comment:
just out of curiosity: is there any packager outputting manifests with decimal resolution?

Reply:
I had some example files I was testing with that had `1280.0x720.0` but I'm not sure where they came from. `ffmpeg` [outputs them as integers](https://github.com/FFmpeg/FFmpeg/blob/e38092ef9395d7049f871ef4d5411eb410e283e0/libavformat/hlsplaylist.c#L86) and I assume that most do as well

Reply:
no worries, thank you